lua/virt-column/init.lua

```diff
@@ -92,6 +93,13 @@ local init = function()
                         })
                     end
                 end
+                local fold_end = vim.api.nvim_win_call(win, function()
+                    return vim.fn.foldclosedend(i)
```

Comment:
```suggestion
                    ---@diagnostic disable-next-line: redundant-return-value
                    return vim.fn.foldclosedend(i)
```

Looks like the types for stable are not correct here :thinking: 
Can you add this to ignore the error?

Reply:
Of course sure! Had commited.